### ISSUE TITLE
Show more instead of pagination

### ISSUE THREAD

**Issue by Giorgi625** [JoomCoder-com/JoomCCK#214]
Is it possible to add show more button at the bottom of the list and allow to display more articles on same page after clicking this button instead of old style pagination?
It would be cool to have this feature as an option in list template to select which you prefer to use.

Of course it is just suggestion for a future, now we have other more important tasks to do.

**Comment by andiamine** [comment]
ajax readmore ?

**Comment by Giorgi625** [comment]
Yes, for example here are reviews of clients https://cleanon.ru/otzyvy/ and at the end of it there is link with dashed underline, above womens had, which shows more reviews.

**Comment by Giorgi625** [comment]
Cool, about my private message, in support tickets, I will let you know after I get contest results, so we can discuss terms of custom work.
